libs/portals/my-pages/law-and-order/src/screens/Verdict/Verdict.tsx

```diff
@@ -0,0 +1,175 @@
+import {
+  LawAndOrderAppealDecision,
+  LawAndOrderItemType,
+} from '@island.is/api/schema'
+import { Divider, toast } from '@island.is/island-ui/core'
+import { useLocale, useNamespaces } from '@island.is/localization'
+import {
+  DOMSMALARADUNEYTID_SLUG,
+  InfoLine,
+  IntroWrapper,
+  m,
+  Modal,
+} from '@island.is/portals/my-pages/core'
+import { Problem } from '@island.is/react-spa/shared'
+import { useMemo, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { AppealForm } from '../../components/VerdictInfoLines/AppealForm'
+import VerdictInfoLines from '../../components/VerdictInfoLines/VerdictInfoLines'
+import { messages } from '../../lib/messages'
+import {
+  useGetCourtCaseVerdictQuery,
+  useSubmitVerdictAppealDecisionMutation,
+} from './Verdict.generated'
+
+type UseParams = {
+  id: string
+}
+
+const CourtCaseDetail = () => {
+  useNamespaces('sp.law-and-order')
+  const { formatMessage, lang } = useLocale()
+  const { id } = useParams() as UseParams
+  const [verdictPopUp, setVerdictPopUp] = useState(false)
+
+  const { data, loading, error } = useGetCourtCaseVerdictQuery({
+    variables: {
+      input: {
+        caseId: id,
+      },
+      locale: lang,
+    },
+    skip: !id,
+  })
+  const verdict = data?.lawAndOrderVerdict
+  const radioButtonGroup = verdict?.groups?.find((g) =>
+    g.items?.some((item) => item.type === LawAndOrderItemType.RadioButton),
+  )
+  const [
+    submitVerdictAppealDecision,
+    { loading: postLoading, data: postData },
+  ] = useSubmitVerdictAppealDecisionMutation()
+
+  // Use postData if available (after submission), otherwise fall back to verdict data
+  const currentAppealDecision = useMemo(() => {
+    return (
+      postData?.lawAndOrderVerdictPost?.appealDecision ??
+      verdict?.appealDecision
+    )
+  }, [
+    postData?.lawAndOrderVerdictPost?.appealDecision,
+    verdict?.appealDecision,
+  ])
+
+  const currentCanAppeal = useMemo(() => {
+    return (
+      postData?.lawAndOrderVerdictPost?.canAppeal ?? verdict?.canAppeal ?? false
```

Comment:
Returning false here feels iffy. `canAppeal` is a boolean, so returning false here seems like it's stems from organization data, but it's a fallback. Can this be more explicit?